### PR TITLE
BaSP-Mr-167: Update users profile: Members, Admins and SuperAdmins

### PR DESCRIPTION
**Pull request by jramire5**
Updated profiles for users, according to the authentification we implemented.

Members and Admins now can change some of their own information.

### CODE REVIEW

File: src/Components/Members/Profile/index.jsx [hunk 1]
```diff
@@ -152,20 +146,36 @@ function MemberProfile({ match }) {
...
           <div className={styles.buttons}>
-            <Button classNameButton="addButton" text={'Edit'} disabled={disableEdit} />
             <Button
-              classNameButton="deleteButton"
-              action={handleReset}
-              text={'Reset'}
-              disabled={disableEdit}
+              classNameButton="addButton"
+              action={() => setDisableEdit(false)}
+              text="Enable Edition"
```
Comment (santimenendez98) [inline src/Components/Members/Profile/index.jsx]: In this part maybe "Edit" would be better

File: src/Components/Members/Profile/index.jsx [hunk 2]
```diff
@@ -152,20 +146,36 @@ function MemberProfile({ match }) {
               </div>
             ))}
           </div>
+          {!disableEdit && (
+            <div className={styles.buttons}>
+              <Button classNameButton="addButton" text={'Edit'} disabled={disableEdit} />
+              <Button
+                classNameButton="cancelButton"
+                action={() => setDisableEdit(true)}
+                text="Disable Edition"
```
Comment (santimenendez98) [inline src/Components/Members/Profile/index.jsx]: In this part maybe "Cancel" would be better

File: src/Components/Members/Profile/index.jsx [hunk 3]
```diff
@@ -152,20 +146,36 @@ function MemberProfile({ match }) {
               </div>
             ))}
           </div>
+          {!disableEdit && (
+            <div className={styles.buttons}>
+              <Button classNameButton="addButton" text={'Edit'} disabled={disableEdit} />
```
Comment (santimenendez98) [inline src/Components/Members/Profile/index.jsx]: In this part maybe "Confirm" would be better

**Review by santimenendez98 — CHANGES_REQUESTED** [review]

**Review by santimenendez98 — CHANGES_REQUESTED** [review]
In the admin edit form, rename the buttons to the same name in the member edit form fix, and obey the order buttons from the members form.

**Review by santimenendez98 — CHANGES_REQUESTED** [review]
The username should be closer to the logout button

**Review by santimenendez98 — CHANGES_REQUESTED** [review]
The date is in iso format, this should be in normal date format (DD/MM/YYYY).

**Review by josedaniele — APPROVED** [review]

**Review by santimenendez98 — APPROVED** [review]

**Pull request merged**